extensions/amp-story/1.0/amp-story.js

```diff
@@ -583,12 +583,12 @@ export class AmpStory extends AMP.BaseElement {
     for (let i = 0; i < interactiveIds.length; i++) {
       if (idsMap[interactiveIds[i]] === undefined) {
         idsMap[interactiveIds[i]] = 0;
-        continue;
+      } else {
+        user().error(TAG, `Duplicate interactive ID ${interactiveIds[i]}`);
+        const newId = `${interactiveIds[i]}__${++idsMap[interactiveIds[i]]}`;
+        interactiveEls[i].id = newId;
+        interactiveIds[i] = newId;
```

Comment:
Just a note for us to remember that this will change with the PR migrating everything to a new extension